bionic/protocols.py

```diff
@@ -314,6 +317,45 @@ def write(self, array, path):
             np.save(file_, array)
 
 
+dd = import_optional_dependency('dask.dataframe', raise_on_missing=False)
+
+
+class DaskProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values always have the
+    ``dask.dataframe.DataFrame`` type.
+
+    These values will be serialized to .dask.pq files.
```

Comment:
nit: we should probably say that we're serializing to a directory.

Reply:
Adding!